fastchat/serve/inference.py

```diff
@@ -160,31 +160,47 @@ def generate_stream(
                 skip_special_tokens=True,
                 spaces_between_special_tokens=False,
             )
+
+            def maybeStop(output, stop_str):
```

Comment:
please follow this function naming style: https://google.github.io/styleguide/pyguide.html#316-naming